src/ast/ddl.rs

```diff
@@ -351,6 +351,10 @@ pub enum AlterTableOperation {
     ValidateConstraint {
         name: Ident,
     },
+    /// `SET ( storage_parameter [= value] [, ... ] )`
```

Comment:
Could we include a link to the docs where this syntax comes from?

Also I'm wondering whether storageparameter is right terminology here? From what I can tell with the syntax it looks possible that it accepts arbitrary sql option in parenthesis? If so thinking something generic like `SetOptionsParens { options: Vec<SqlOption> }` or similar naming as the enum variant

Reply:
so Postgres does expect one of a specific set of parameters there: https://www.postgresql.org/docs/current/sql-createtable.html#SQL-CREATETABLE-STORAGE-PARAMETERS

and will give `ERROR:  unrecognized parameter` for an unexpected parameter. The parameters also have been the same in the last 5 versions, so they seem relatively stable.

Happy to either change to the more generic form, or update the parsing to look for the specific storage parameters instead of arbitrary options. Whatever you think is more appropriate @iffyio

Reply:
Ah I see, indeed I think the generic form would be preferred beyond the postgres dialect in this case